src/components/Breadcrumbs/index.tsx

```diff
@@ -16,51 +18,58 @@ interface BreadcrumbsProps {
 const Breadcrumbs: React.VFC<BreadcrumbsProps> = ({ breadcrumbs }) => (
   <Container>
     {breadcrumbs.map((bc, index) => (
-      <Fragment
-        breadcrumb={bc}
-        key={`breadCrumb-${bc.text}`}
-        active={breadcrumbs.length - 1 === index}
-      />
+      <Fragment key={`breadCrumb-${bc.text}`}>
+        <BreadcrumbFragment breadcrumb={bc} />
+        {breadcrumbs.length - 1 !== index && (
+          <Icon data-cy="breadcrumb-chevron" glyph="ChevronRight" />
```

Comment:
I think adding some spacing by styling the icon with `margin: 0 ${size.xxs};` would be nice